lib/pythonx.ex

```diff
@@ -290,6 +290,9 @@ defmodule Pythonx do
       for name <- defined do
         quote do
```

Comment:
After taking a look at Elixir's source, I also think we can get the same behaviour by marking it as generated:

    quote generated: true do